hackerrank/src/Algorithms/Warmup/CompareTriplets.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Warmup.CompareTriplets;
+
+public class Result
+{
+    public static List<int> compareTriplets(List<int> a, List<int> b)
+    {
+        List<int> result = new List<int>() { 0, 0 };
+
+        for (int i = 0; i <= 2; i++)
```

Comment:
Хардкод не вельмі добра.
```suggestion
        for (int i = 0; i < a.Count; i++)
```